src/forms/renderers/OAuth.tsx

```diff
@@ -185,6 +202,36 @@ const OAuthproviderRenderer = ({
         }
     };
 
+    useEffect(() => {
+        const customErrors = [];
+
+        // Used to set an error for the OAuth Renderer
+        //  Check if there are already errors related to the renderer
+        //  so we do not end up with redundant error messages.
+        if (!hasAllRequiredProps && !hasLength(errors)) {
+            customErrors.push({
+                instancePath: path,
+                message: `must have required property 'OAuth'`,
```

Comment:
I'm not sure what this error is signifying, but we don't have a property called 'OAuth' anywhere I think 🤔

Reply:
Initially I had this saying "You need to authenticate" but looks kinda weird since all the other messages are just "must have required property"